### PR TITLE
ADD: Dark mode support and Clear button for Is it my address

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: loc/en.json [hunk 1]
```diff
@@ -481,7 +481,8 @@
     "title": "Is it my address?",
     "owns": "{label} owns {address}",
     "enter_address": "Enter address:",
-    "check_address": "Check address"
+    "check_address": "Check address",
+    "no_wallet_owns_address": "None of the available wallets own the provided address.",
```
Comment (Overtorment) [inline loc/en.json:485]: broken json

File: loc/en.json [hunk 2]
```diff
@@ -481,7 +481,8 @@
     "title": "Is it my address?",
     "owns": "{label} owns {address}",
     "enter_address": "Enter address:",
-    "check_address": "Check address"
+    "check_address": "Check address",
```
Comment (Overtorment) [inline loc/en.json:484]: btw lets make `enter_address` a placeholder in text input instead of standalone text

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by Overtorment** [comment]
image

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/nwh5t1qc396w6f04kv6cr8fpzr

[download apk](http://filestorage.bluewallet.io:1488/1607628405.apk)

**Comment by Overtorment** [comment]
conflict

**Comment by marcosrdz** [comment]
> conflict

resolved

**Review by Overtorment — CHANGES_REQUESTED** [review]
linting failed

**Comment by marcosrdz** [comment]
> linting failed

resolved

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7vjyc2xf7m8gnfnd0rx8r5ytpw

[download apk](http://filestorage.bluewallet.io:1488/1607727290.apk)

**Pull request merged**